0035-ilp-over-http/0035-ilp-over-http.md

```diff
@@ -32,6 +32,7 @@ Host: example.com
 Accept: application/octet-stream
 Content-Type: application/octet-stream
 Authorization: Bearer zxcljvoizuu09wqqpowipoalksdflksjdgxclvkjl0s909asdf
+ILP-Account-Name: alice
```

Comment:
Can we call this `ILP-Account-Id` instead?

Reply:
I prefer the id suffix over name. I know @adrianhopebailie was a fan of using `ilp-peer-id`. Do we have a consensus on `peer` vs `account`

Reply:
I think we should use `account` here, but perhaps it's a naming ambiguity thing. If `peer` == `account`, then it doesn't matter. 

However, in my mind, `peer` != `account` because a peer could have multiple accounts potentially (e.g., imagine you and me are peered with one credential that can operate across currencies/accounts).

However, in ILP, I don't think we should introduce that type of peering because it's much simpler to just have accounts. So, I vote for `account` and we can (maybe never) use the `peer` term in the future.

Reply:
> But, in this context I think we are talking about a peer not an account
Can you clarify why? I'd like to _not_ introduce peering into ILP if we can help it. In my mind it's actually simpler and more secure to only have accounts, with each account using a different auth credential.